tests/zeroconfig/windows/windows_iis_test.go

```diff
@@ -86,17 +86,15 @@ func TestWindowsIISInstrumentation(t *testing.T) {
 		return resp.StatusCode == http.StatusOK
 	}, 30*time.Second, 100*time.Millisecond)
 
-	// Make a GET request to the ASP.NET app on the Windows IIS container
-	httpClient := &http.Client{}
-	req, err := http.NewRequest("GET", "http://localhost:8000/aspnetfxapp/api/values", nil)
-	require.NoError(t, err)
-	resp, err := httpClient.Do(req)
-	require.NoError(t, err)
-	defer resp.Body.Close()
-	require.Equal(t, http.StatusOK, resp.StatusCode)
+	requireHTTPGetRequestSuccess(t, "http://localhost:8000/aspnetfxapp/api/values/4")
 	t.Log("ASP.NET HTTP Request succeeded")
+	expectedResourceTraces, err := telemetry.LoadResourceTraces("expected_aspnetfxapp_traces.yaml")
```

Comment:
There's yet to be `testutils.Testcase` helper adoption but there's a convention to have a `testdata/resource_<type>` directory where [the loader takes care of some validation](https://github.com/signalfx/splunk-otel-collector/blob/main/tests/testutils/testcase.go#L110). Until a better option system is implemented that more readily accepts arbitrary otlp sink endpoints there's not a real need for it in this suite imo.

In the meantime do you mind moving these to something like `testdata/resource_spans/aspnetfx.yaml` and `testdata/resource_spans/aspnetcore.yaml` for consistency?

Reply:
just for consistency with the type I will use `testdata/resource_traces` (that issue that pdata uses "traces").

Reply:
Fixed via 7de808941ab7e67e7c0c0319764454c3a9a5ddd0